.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
Let's not commit this file

Reply:
Re: removing this and the extensions.json --- understood, will remove these.

Is there a reason to include `.vscode/c_cpp_properties.json`? If not I can remove that from repo and add `.vscode` to `.gitignore`.

I can add these all to future documentation as suggestions for dev workflow?

Reply:
Let's remove that and the `.vscode` folder also

Reply:
Sounds good. I saw timescale has `/.vscode` in `.gitignore`